modules/go-health-monitoring.adoc

```diff
@@ -17,7 +17,7 @@ The {gitops-title} Operator will install the GitOps backend service in the `open
 
 . Click the application name from the list to view the details of a specific application.
 
-. If the *Resources* section displays icons, hover over the icons to get status details. 
+. In the *Application environments* page, if the *Resources* section under the *Overview* tab displays icons, hover over the icons to get status details. 
```

Comment:
I see you introduced "Application environments" page for the first time, but the same page was called "environment details page" on the same documentation. Please refer the first paragraph on this link: https://docs.openshift.com/container-platform/4.12/cicd/gitops/health-information-for-resources-deployment.html.
It makes more sense if we keep it consistent across the same documentation.

Reply:
Hi, @ciiay: I have done the changes that were needed. PTAL at the revised PR and provide feedback.

As part of implementing your suggestion, I will be requesting @shipsing to make the necessary changes in [this PR](https://github.com/openshift/openshift-docs/pull/56052) as well. I will sync up with her tomorrow for this and ensure that the required changes are captured in the RN documentation as well. That way, the terms will be consistent throughout the OCP GitOp docs.